srcpkgs/exult/template

```diff
@@ -0,0 +1,41 @@
+# Template file for 'exult'
+pkgname=exult
+version=1.10.1
+revision=1
+archs="i686 x86_64"
+hostmakedepends="automake autoconf autoconf-archive libtool pkg-config fluidsynth-devel libmt32emu-devel libogg-devel libpng-devel libvorbis-devel SDL2-devel zlib-devel"
+depends="fluidsynth libmt32emu libogg libpng libvorbis SDL2 zlib"
```

Comment:
those dependencies are all detected through the linked shared libraries and can be removed here. (https://github.com/void-linux/void-packages/actions/runs/15095098726/job/42428719230?pr=54516#step:7:1401)